ui/infra-compare/InfraCompareTableRow.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class InfraCompareTableRow extends React.PureComponent {
+  getColorClass = (data, type) => {
+    const { className, isRegression, isImprovement } = data;
+    if (type === 'bar' && !isRegression && !isImprovement) return 'secondary';
+    if (type === 'background' && className === 'warning')
+      return `bg-${className}`;
+    if (type === 'text' && className) return `text-${className}`;
+    return className;
+  };
+
+  render() {
+    const { rowLevelResults } = this.props;
+    return (
+      <React.Fragment>
```

Comment:
Let's add a blank line above the `return`.  Also, no need for this `Fragment` as well.